project/make/cross

```diff
@@ -28,6 +30,21 @@ for platform in $DOCKER_CROSSPLATFORMS; do
 			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
 			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
 		fi
+		if [ $GOARCH = "arm" ] && [ $GOOS = "linux" ]
+		then
+			export CC=arm-linux-gnueabi-gcc
+			export LD_FLAGS="-extld=arm-linux-gnueabi-gcc"
+			export CGO_ENABLED=1
+			export LIBRARY_PATH=$LIBRARY_PATH:/usr/arm-linux-gnueabi/lib
```

Comment:
Agreed @tianon - Didn't think of this - How about we create a new make target like so 
```
make cross-build
```
and move the changes which cross-compile there?